src/components/game-rules/game-rules.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { type FC } from "react";
+import { Typography } from "@mui/material";
+
+import { text as copywrite } from "~/assets/text";
```

Comment:
Remember to add the file extension.
We definitely need an extra linting rule for this.